packages/fxa-admin-panel/package.json

```diff
@@ -73,11 +83,13 @@
     "@types/react-router-dom": "^5.1.3",
     "@types/serve-static": "^1.13.3",
     "@types/supertest": "^2.0.8",
+    "@types/webpack": "4.41.16",
     "@typescript-eslint/eslint-plugin": "^2.21.0",
     "@typescript-eslint/parser": "^2.21.0",
     "apollo-cache-inmemory": "^1.6.2",
     "apollo-link": "^1.2.12",
     "apollo-utilities": "^1.3.2",
+    "babel-loader": "^8.1.0",
```

Comment:
where's this used?

Reply:
Storybook makes use of this and webpack (as well as @babel/core), yarn complains when they're not present. These are present in fxa-settings, fxa-payments-server as well.